tests/acceptance/features/bootstrap/WebUITagsContext.php

```diff
@@ -103,6 +104,18 @@ public function theUserTypesAValueInTheCollaborativeTagsFieldUsingTheWebUI($valu
 		$this->filesPage->getDetailsDialog()->insertTagNameInTheTagsField($value);
 	}
 
+	/**
+	 * @When the user edits the tag with name :oldName and sets its name to :newName using the webUI
```

Comment:
```suggestion
	 * @When the user renames tag  :oldName  to :newName using the webUI
```

This is just an suggestion, pick which one you think is better.